apps/condo/domains/banking/components/BankAccountVisibilitySelect.tsx

```diff
@@ -35,22 +38,23 @@ export const BankAccountVisibilitySelect: IBankAccountVisibilitySelect = ({ bank
     const accountVisibility = get(selectedBankAccount, 'publishedAt', null)
 
     const [isUpdating, setIsUpdating] = useState(false)
-    const reportVisibleRef = useRef<BankAccountVisibility>(Number(!isNull(accountVisibility)))
+    const [reportVisible, setReportVisible] = useState<BankAccountVisibility>(Number(!isNull(accountVisibility)))
```

Comment:
`isReportVisible`, please user `is` prefix for boolean values for better understandability